docs/source/secrets.rst

```diff
@@ -1,33 +0,0 @@
-Secrets
```

Comment:
@mime-frame secrets are transparently sent kms/s3 behind the scenes and the process outlined `outputs.html#configuration` walks the user through adding the output.

As simple as:
`python stream_alert_cli.py output new --service slack`

Which then prompts for things like channel name and integration key (the latter of which is sent to kms/s3 for encryption/storage).